CL/cl.h

```diff
@@ -846,6 +846,9 @@ typedef struct _cl_buffer_region {
 #define CL_PROFILING_COMMAND_COMPLETE               0x1284
 #endif
 
+/* cl_khronos_vendor_id */
+#define CL_KHRONOS_VENDOR_ID_CODEPLAY               0x10004
```

Comment:
Not sure about the overall ordering of defines, there may be a more appropriate location for this